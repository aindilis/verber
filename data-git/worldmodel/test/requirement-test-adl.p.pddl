(define
 (problem REQUIREMENT-TEST-ADL)
 (:domain REQUIREMENT-TEST-ADL)
 (:objects obj2 obj1 - type1)
 (:init
  (pred1 obj1))
 (:goal
  (and
   (pred1 obj2))))