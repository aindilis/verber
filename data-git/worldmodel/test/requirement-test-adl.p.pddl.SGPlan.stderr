

Starting SGPlan search ...

Solution found.
Answer file is /var/lib/myfrdcsa/codebases/internal/verber/data/worldmodel/test/requirement-test-adl.p.pddl.SGPlan.sol

